shared/src/main/scala/mouse/MouseFunctions.scala

```diff
@@ -0,0 +1,13 @@
+package mouse
+
+trait MouseFunctions {
```

Comment:
Because the package objects are now implemented within each configuration, we needed somewhere to define bare functions that were previously defined on the package object. I've done this with this trait, which needs to be implemented by each of the two package objects.